app/src/main/java/org/oppia/android/app/testing/SplashTestActivityPresenter.kt

```diff
@@ -25,10 +25,9 @@ class SplashTestActivityPresenter @Inject constructor(
 ) {
   fun handleOnCreate() {
     activity.setContentView(R.layout.splash_test_activity)
-    loadPlatformParameters()
   }
 
-  private fun loadPlatformParameters() {
+  fun loadPlatformParameters() {
```

Comment:
Add a KDoc for this function now that you have opened it up!

Reply:
Done